pkgs/development/libraries/libunistring/default.nix

```diff
@@ -14,6 +14,8 @@ stdenv.mkDerivation rec {
 
   propagatedBuildInputs = stdenv.lib.optional (!stdenv.isLinux) libiconv;
 
+  enableParallelBuilding = false;
```

Comment:
I've been having trouble building `libunistring` on openSUSE using nix, which I only solved by using an overlay that disabled tests for that particular library. It always got stuck in the `test-lock` test with full CPU usage. Does this solve that particular issue? I think I had the same problem with `findutils`. Let me see if I can recreate the issue on that particular machine.